src/components/footer.jsx

```diff
@@ -0,0 +1,68 @@
+"use strict";
+
+import React from "react";
+import Router from "react-router";
+import { Navbar, Nav, NavItem, Row, Col, Thumbnail } from "react-bootstrap";
+import { NavItemLink } from "react-router-bootstrap";
+
+let { Route, Link, RouteHandler } = Router; // eslint-disable-line
+
+import "./footer.less";
+import Facebook from "../../assets/img/icons/fb.png";
+import Twitter from "../../assets/img/icons/twitter.png";
+import Google from "../../assets/img/icons/gplus.png";
+import Github from "../../assets/img/icons/github.png";
+
+export default React.createClass({
+  displayName: "Footer",
+
+  render() {
+    return <Row className="footer">
+        <Col xs={12} md={12} className="block">
```

Comment:
Nu trebuie neaparat sa pui `xs` si `md`. Lasă doar `md` că doar pentru asta avem design-ul făcut.

http://stackoverflow.com/questions/19865158/what-is-the-difference-among-col-lg-col-md-and-col-sm-in-twitter-bootstra